libcarina.go

```diff
@@ -442,3 +447,19 @@ func (c *ClusterClient) Delete(clusterName string) (*Cluster, error) {
 	resp, err := c.NewRequest("DELETE", uri, nil)
 	return clusterFromResponse(resp, err)
 }
+
+func (c *ClusterClient) GetQuotas() (*Quotas, error) {
+	uri := path.Join("/quotas", c.Username)
+	resp, err := c.NewRequest("GET", uri, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	quotas := new(Quotas)
```

Comment:
Having said that, I don't mind doing it if you think there'll be a use for it in the future. I will be pushing more to this as I add some testing stuff, anyway.